src/operatingSystem.ts

```diff
@@ -0,0 +1,21 @@
+/** @module operatingSystem */
+
+/** @constant {string} */
+const UNKNOWN = 'unknown';
+
+/** @constant {string} */
+const WINDOWS = 'win32';
+
+/** @constant {string} */
+const MACOS = 'darwin';
+
+/** @constant {string} */
+const LINUX = 'linux';
+
+/** @constant {string} */
+const ANY = 'any';
+
+/** @constant {string[]} */
+const operatingSystems = [UNKNOWN, WINDOWS, MACOS, LINUX, ANY];
+
+export { UNKNOWN, WINDOWS, MACOS, LINUX, ANY, operatingSystems };
```

Comment:
Similarly this would look like:
```suggestion
export enum OperatingSystem {
  Unknown = 'unknown',
  Windows = 'win32',
  MacOS = 'darwin',
  Linux = 'linux',
  Any = 'any
}
```